downstream/modules/platform/proc-aap-controller-backup.adoc

```diff
@@ -18,8 +18,7 @@ Use this procedure to back up a deployment of the controller, including jobs, in
 . Select the *Automation Controller Backup* tab.
 . Click btn:[Create AutomationControllerBackup].
 . Enter a *Name* for the backup.
-. Enter the *Deployment name* of the deployed {PlatformNameShort} instance being backed up.
-For example, if your {ControllerName} must be backed up and the deployment name is `aap-controller`, enter 'aap-controller' in the *Deployment name* field.
+. Enter the *Deployment name* of the deployed {PlatformNameShort} instance being backed up. The deployment name displays in the following format, `<deployment_name>-[web|task]`. For example, if your {ControllerName} must be backed up and the deployment name is 'automationcontroller-2b5a30e4-web` or `automationcontroller-2b5a30e4-task', enter `automationcontroller` in the *Deployment name* field to filter the list down to applicable deployments.
```

Comment:
Almost ...

format, `<deployment_name>-[web|task]` --> format, `<deployment_name>-<random_hash>-[web|task]`

`enter `automationcontroller` in the *Deployment name*` --> `automationcontroller-2b5a30e4`

Reply:
@chrismeyersfsu  how did they know what their random hash is?

Reply:
They don't/won't. I didn't. I kind of guessed since it was the only one. It's possible they have multiple but that would mean they have multiple Controllers installed in the same namespace, which users _probably_ won't have.

Reply:
@chrismeyersfsu - how would they enter it in their deployment name if they don't know what their <random_hash> is?

Reply:
I might be totally wrong on this topic @tvo318 . I'd like @shanemcd to step in here and :eyes:

Reply:
@shanemcd has explained that it's not the _Deployment name_ you are entering here. It’s the name of the AutomationController custom resource object.